src/main/java/bio/terra/service/filedata/google/firestore/FireStoreDirectoryDao.java

```diff
@@ -200,7 +199,7 @@ private FireStoreDirectoryEntry runTransactionWithRetry(Firestore firestore,
                                                             LookupFunction lookupFunction,
                                                             String transactionOp,
                                                             String warnMessage) throws InterruptedException {
-        for (int i = 0; i < RETRIES; i++) {
+        for (int i = 0; i < LOOKUP_RETRIES; i++) {
```

Comment:
This should fix a second bug we were running into - We were getting some timeouts on file ingest, but hopefully bumping the number of retries up should help.